doc/classes/Spatial.xml

```diff
@@ -110,14 +110,14 @@
 			<return type="bool">
 			</return>
 			<description>
-				Returns whether this node is set as Toplevel, ignoring its parent node transformations.
+				Returns whether this node is set as Toplevel (whether ignores its parent nodes transformations).
```

Comment:
I don't understand the parenthesis. Do you mean something like:
"Returns whether this node is set as "top level", i.e. it ignores its parent nodes' transformations."

Reply:
Yup. "i.e." is better idea tho. Will update.